src/api/app/policies/staging_workflow_policy.rb

```diff
@@ -0,0 +1,11 @@
+class StagingWorkflowPolicy < ApplicationPolicy
```

Comment:
[nitpick] Can we not just inherit from ProjectPolicy?

Reply:
No, because it would have more methods than what are needed

Reply:
Ok, fine for me.

We should at some point really extract a Policy class with optional user ... We repeat this in every second Policy :scream: